submissions/andysmokk/classic-frogger-game/app.js

```diff
@@ -0,0 +1,105 @@
+// Enemies our player must avoid
+const POSITION_PLAYER_X = 203;
+const POSITION_PLAYER_Y = 390;
+const POSITION_ENEMIES_X = -200;
+const BLOCK_HEIGHT = 83;
+const BLOCK_WIDTH = 101;
+const MIN_SPEED = 100;
+const MAX_SPEED = 400;
+
+var Enemy = function (x, y, user) {
+  this.x = x;
+  this.y = y;
+  this.user = user;
+  this.speed = this.randomInteger();
+  this.sprite = "images/enemy-bug.png";
+};
+
+Enemy.prototype.update = function (dt) {
+  this.x += this.speed * dt;
+
+  if (this.x > 500) {
```

Comment:
What is 500?

Reply:
Is the wigth of the playing field. Fixed.